modules/docker-registry.nix

```diff
@@ -45,16 +51,79 @@ in {
         a basic-auth file for registry authentication.
       '';
     };
+
+    enableRepair = mkOption {
+      type = bool;
+      default = true;
+      description = "Enables the docker registry repair service.";
+    };
+
+    repairDeleteTag = mkOption {
+      type = bool;
+      default = false;
+      description = "Also delete all tag references during repair.";
+    };
+
+    repairDryRun = mkOption {
+      type = bool;
+      default = false;
+      description = "Avoid deleting anything during repair.";
+    };
+
+    repairPkg = mkOption {
+      type = package;
+      default = pkgs.docker-registry-repair;
+      description = ''
+        The reigstry repair package to utilize.
```

Comment:
```suggestion
        The registry repair package to utilize.
```

Reply:
Thanks!  Also, yes, the `sleep': can't convert String into time interval (TypeError)` above was fixed before these commits were pushed.